### PR TITLE
Migrate content from `meta/` to the team compass

### PR DESCRIPTION
**Pull request by choldgraf**
This PR migrates internal team documentation from `meta/` to the team compass. It tries to pull in any information that is "more for the team than it is for the external world". It'll be paired with a PR that removes this content from `meta/` as well.

### References

- closes https://github.com/executablebooks/meta/pull/770 by superceding it

### CODE REVIEW

**Review by gregcaporaso — APPROVED** [review]
This looks good to me @choldgraf, thank you!

File: docs/about/history.md [hunk 1]
```diff
@@ -2,8 +2,10 @@
 
 This is a brief history of the Executable Books project, written with the intention of encoding the provenance and rationale behind our current governance structure.
 
-The Executable Books project began as a collaboration between three universities funded from a single grant.
-This team used these resources to develop a collection of tools and standards for communicating ideas with computational narratives in the Jupyter ecosystem and beyond.
+The Executable Books project began as a grant-funded collaboration between groups at [The Australian National University](https://anu.edu.au),
+[Northern Arizona University](https://nau.edu/), and
+[The University of California at Berkeley](https://www.berkeley.edu/).
+These teams represented many open source projects in the scientific community, such as [QuantEcon](https://quantecon.org), [QIIME](https://qiime2.org/), and [The Jupyter Project](https://jupyter.org/).
```
Comment (choldgraf) [inline docs/about/history.md:8]:
```suggestion
These teams represented many open source projects in the scientific community, such as [QuantEcon](https://quantecon.org), [QIIME 2](https://qiime2.org/), and [The Jupyter Project](https://jupyter.org/).
```

File: docs/about/team.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
-# Team Members
+(team)=
+# Team members
 
+This project has had contribution from many different people and organizations.
```
Comment (choldgraf) [inline docs/about/team.md:4]:
```suggestion
This project has had contributions from many different people and organizations.
```

**Comment by choldgraf** [comment]
I've addressed your suggestions @gregcaporaso - also made some minor naming changes and updates to avoid confusion.

- The `conventions.md` file now `{include}`s the file at the `.github/` repository and we have an explanation for how this works to make it easier to follow.

**Comment by choldgraf** [comment]
I'm gonna merge this one in since we've already got a +1 from @gregcaporaso and can iterate from there

**Pull request merged**